docOps/docs/libraries/python/examples.md

```diff
@@ -0,0 +1,756 @@
+
+All the examples in this section expect you to set your custom password  in the _.env_ file:
+
+```bash
+SH_PASSWORD="here is your super secure password"
+```
+
+## Account Manager and Individual Accounts
+You can initialize (open) a secure storage for individual accounts.  The storage is backed up by `Stronghold` by default, using an AccountManager instance.  
+
+The following example creates a new database and account:
+```python
+# Copyright 2020 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+
+import iota_wallet as iw
+import os
+from dotenv import load_dotenv
+
+
+# Load the env variables
+load_dotenv()
+
+# Get the Stronghold password
+STRONGHOLD_PASSWORD = os.getenv('STRONGHOLD_PASSWORD')
+
+account_manager = iw.AccountManager(
+    storage_path='./alice-database'
+)  # note: `storage` and `storage_path` have to be declared together
+
+account_manager.set_stronghold_password(STRONGHOLD_PASSWORD)
+
+# mnemonic (seed) should be set only for new storage
+# once the storage has been initialized earlier then you should omit this step
+account_manager.store_mnemonic("Stronghold")
+```
+* Storage is initialized under the given path (`./alice-database`)
+* The password is set based on your password in _.env_ file ( _manager.setStrongholdPassword(process.env.SH_PASSWORD)_ )
+* When you initialize the new database, a Stronghold mnemonic (seed) is automatically generated and stored by default ( _manager.storeMnemonic(SignerType.Stronghold)_ ).
+* The seed should be set only for the first time. In order to open already initialized database, you can simply use your password.
+
+The storage is encrypted at rest, so you need a strong password and location where to place your storage. 
+
+:::warning
+We highly recommended that you to store your `Stronghold` password encrypted on rest and separated from `Stronghold` snapshots. 
+
+Deal with the password with utmost care.
+:::
+
+ The storage comprises two things:
+* A single file called _wallet.stronghold_ , which contains _seed_ and is secured by `Stronghold` and encrypted at rest. The generated seed (mnemonic) serves as a cryptographic key from which all accounts and related addresses are generated.
+* Other data used by library that is stored under the _db_ sub-directory.  The includes account information, generated addresses, fetched messages, etc.  This data is used to speed up some operations, such as account creation, address generation, etc.
+
+One of the key principles behind `Stronghold` based storage is that no one can extract a seed from the storage. You deal with all accounts purely via an _AccountManager_ instance.  All complexities are hidden under the hood and dealt with securely.
+
+If you also want to store a seed somewhere else, you can use the _AccountManager.generateMnemonic()_ method. This method will generate a random seed, and it can be used before the actual account initialization.
```

Comment:
```suggestion
If you also want to store a seed somewhere else, you can use the `AccountManager.generateMnemonic()` method. This method will generate a random seed, and it can be used before the actual account initialization.
```